server/ai_process.go

```diff
@@ -1098,10 +1101,17 @@ func submitLiveVideoToVideo(ctx context.Context, params aiRequestParams, sess *A
 			monitor.AIFirstSegmentDelay(delayMs, sess.OrchestratorInfo)
 		}
 		clog.V(common.VERBOSE).Infof(ctx, "First Segment delay=%dms streamID=%s", delayMs, params.liveParams.streamID)
+		firstSegmentReceived <- struct{}{}
```

Comment:
Might need a select with default in case timeout has already fired, or a buffered chan

Reply:
Changed to buffered chan